### PR TITLE
Updated for changes from recent update

### PR DESCRIPTION
**Pull request by BaileyLawson**
Updated for changes from recent update

### CODE REVIEW

File: articles/azure/azs-how-ps-update-azure-stack.md [hunk 1]
```diff
@@ -35,30 +35,25 @@ Prerequisites from a Windows-based external client.
...
-3. Upload the packages to Azure Stack.
+2. Go to the **Update** blade on the Administration Portal
 
-4. Start Update process.
+3. Start Update process.
 
 > [!NOTE]
 > Automatic Update Guide to follow
 >
 > Code for Automatic Update can be found here -> [more](https://github.com/UKCloud/AZS_dev_black/tree/master/docs/code/powershell/Update)
```
Comment (suehighmoor) [inline articles/azure/azs-how-ps-update-azure-stack.md]: Link doesn't work

File: articles/azure/azs-how-ps-update-azure-stack.md [hunk 2]
```diff
@@ -35,30 +35,25 @@ Prerequisites from a Windows-based external client.
...
 
-# Automated Update Guide
+## Feedback
 
-1. 
+If you have any comments on this document or any other aspect of your UKCloud experience, send them to <products@ukcloud.com>.
 
-(c) UKCloud Ltd, 2018. All Rights Reserved.
\ No newline at end of file
+(c) UKCloud Ltd, 2018. All Rights Reserved.
```
Comment (suehighmoor) [inline articles/azure/azs-how-ps-update-azure-stack.md]: Remove this line

File: articles/azure/azs-how-ps-update-azure-stack.md (hunk 1 above)
Comment (BaileyLawson) [reply articles/azure/azs-how-ps-update-azure-stack.md]: Fixed

File: articles/azure/azs-how-ps-update-azure-stack.md (hunk 2 above)
Comment (BaileyLawson) [reply articles/azure/azs-how-ps-update-azure-stack.md]: Fixed

File: articles/azure/azs-how-ps-update-azure-stack.md [hunk 3]
```diff
@@ -35,30 +35,23 @@ Prerequisites from a Windows-based external client.
...
 
 > [!IMPORTANT]
 >
->
 > We strongly recommend that you notify users of any maintenance operations, and that you schedule normal maintenance windows during non-business hours as much as possible. Maintenance operations may affect both user workloads and portal operations.
 
 1. Raise a pre-emptive case with Microsoft prior to starting the Update.
 
-2. Download required packages.
-* Full Manual procedure can be found here -> [more](https://docs.microsoft.com/en-us/azure/azure-stack/azure-stack-apply-updates#import-and-install-updates)
```
Comment (suehighmoor) [inline articles/azure/azs-how-ps-update-azure-stack.md]: You can find the full manual procedure [here](https://docs.microsoft.com/en-us/azure/azure-stack/azure-stack-apply-updates#import-and-install-updates).

File: articles/azure/azs-how-ps-update-azure-stack.md [hunk 4]
```diff
@@ -35,30 +35,23 @@ Prerequisites from a Windows-based external client.
...
+## High Level Overview of the process
 
 > [!IMPORTANT]
 >
->
 > We strongly recommend that you notify users of any maintenance operations, and that you schedule normal maintenance windows during non-business hours as much as possible. Maintenance operations may affect both user workloads and portal operations.
 
 1. Raise a pre-emptive case with Microsoft prior to starting the Update.
 
-2. Download required packages.
```
Comment (suehighmoor) [inline articles/azure/azs-how-ps-update-azure-stack.md]: Download the required packages.

File: articles/azure/azs-how-ps-update-azure-stack.md [hunk 5]
```diff
@@ -35,30 +35,23 @@ Prerequisites from a Windows-based external client.
...
-* Full Manual procedure can be found here -> [more](https://docs.microsoft.com/en-us/azure/azure-stack/azure-stack-apply-updates#import-and-install-updates)
-  * Download link -> [download](https://aka.ms/azurestackupdatedownload)
-
-3. Upload the packages to Azure Stack.
+2. Go to the **Update** blade on the Administration Portal
```
Comment (suehighmoor) [inline articles/azure/azs-how-ps-update-azure-stack.md]: Full stop at the end

File: articles/azure/azs-how-ps-update-azure-stack.md [hunk 6]
```diff
@@ -35,30 +35,23 @@ Prerequisites from a Windows-based external client.
...
 
-4. Start Update process.
+3. Start Update process.
```
Comment (suehighmoor) [inline articles/azure/azs-how-ps-update-azure-stack.md]: Start the update process.

File: articles/azure/azs-how-ps-update-azure-stack.md [hunk 7]
```diff
@@ -35,30 +35,23 @@ Prerequisites from a Windows-based external client.
...
 
 > [!NOTE]
 > Automatic Update Guide to follow
 >
-> Code for Automatic Update can be found here -> [more](https://github.com/UKCloud/AZS_dev_black/tree/master/docs/code/powershell/Update)
-
-# Automated Update Guide
+> Code for Automatic Update can be found here -> [more](https://github.com/UKCloud/AZS_dev_black/tree/master/docs/code/powershell/UpdateManagement)
```
Comment (suehighmoor) [inline articles/azure/azs-how-ps-update-azure-stack.md]: You can find the code for automatic updates [here](https://github.com/UKCloud/AZS_dev_black/tree/master/docs/code/powershell/UpdateManagement).

File: articles/azure/azs-how-ps-update-azure-stack.md (hunk 5 above)
Comment (BaileyLawson) [reply articles/azure/azs-how-ps-update-azure-stack.md]: Fixed

File: articles/azure/azs-how-ps-update-azure-stack.md (hunk 6 above)
Comment (BaileyLawson) [reply articles/azure/azs-how-ps-update-azure-stack.md]: Fixed

File: articles/azure/azs-how-ps-update-azure-stack.md (hunk 7 above)
Comment (BaileyLawson) [reply articles/azure/azs-how-ps-update-azure-stack.md]: Fixed

**Pull request merged**